runtime/wasmer_test.go

```diff
@@ -651,3 +651,60 @@ func TestExt_twox_128(t *testing.T) {
 		t.Error("hash saved in memory does not equal calculated hash")
 	}
 }
+
+// test ext_malloc returns expected pointer value of 8
+func TestExt_malloc(t *testing.T) {
```

Comment:
maybe we can add more tests for malloc/free for various sizes?

Reply:
refactored and added tests to `allocator_test.go`